package.json

```diff
@@ -23,9 +23,9 @@
     "predev": "npm run build",
     "dev": "concurrently \"npm run dev --workspace @govuk-frontend/review\" \"npm run dev --workspace govuk-frontend\" --kill-others --names \"app,pkg\" --prefix-colors \"red.dim,blue.dim\"",
     "build": "npm run build --workspace govuk-frontend --workspace @govuk-frontend/review",
-    "build:app": "npm run build --workspace @govuk-frontend/review",
-    "build:package": "npm run build:package --workspace govuk-frontend",
-    "build:release": "npm run build:release --workspace govuk-frontend",
+    "build:app": "cross-env-shell NODE_ENV=production npm run build --workspace @govuk-frontend/review",
```

Comment:
I think this is probably fine as when e.g. doing the build for a release we _should_ be following the process in our release docs, but wondering if it's worth trying to flip this so that we 'default to' creating a production build and the review app opts out?

Would that be the safer behaviour?

Reply:
I was very torn by adding this

Problem came from PostCSS following this Node.js principle:
https://nodejs.dev/en/learn/nodejs-the-difference-between-development-and-production/

>Node.js assumes it's always running in a development environment. You can signal Node.js that you are running in production by setting the `NODE_ENV=production` environment variable.

So it would be safer for sure, but we're not in control of the defaults our dependencies us 😣

E.g. If we assume `production` as the default, what if CSSNANO defaults to `development`?

Reply:
Yep, that's fair. We'll just have to be careful to make sure we're using the top-level build scripts (or setting the env) when doing anything where it's important we have the version number etc.